README.md

```diff
@@ -1,19 +1,38 @@
 # CNCF Mentoring Initiatives
 
-The Cloud Native Computing Foundation (CNCF) participates in a variety of mentoring programs. CNCF is a great place to spend a summer learning, coding, participating, and contributing. We are an exciting open source foundation with a vibrant community of projects, and we look forward to your application and your project ideas!
+The Cloud Native Computing Foundation (CNCF) participates in a variety of mentoring programs. CNCF is a great place to spend a summer learning, coding, documenting, participating, and contributing. We look forward to your application and your project ideas!
```

Comment:
I wonder if "summer" should be "season" -- Since this is one of the first bits of info folks will see I don't  want them to think that the programs only run in summer.

```suggestion
The Cloud Native Computing Foundation (CNCF) participates in a variety of mentoring programs. CNCF is a great place to spend a season learning, coding, documenting, participating, and contributing. We look forward to your application and your project ideas!
```

Reply:
Agree, updated